src/entity/Student.java

```diff
@@ -0,0 +1,55 @@
+package entity;
+import java.util.List;
+import java.util.Objects;
+
+public class Student implements Cloneable {
```

Comment:
nu sunt sigura ca inteleg aici, de ce ai implementat Cloneable?

Reply:
cand incercam sa fac sortare cu comparator, am incercat intai cu Cloneable, apoi am facut cu Comparator, insa am uitat sa sterg acel "implements Cloneable". It's done now, nu au aparut erori sau alte modificari.